weave/trace_server/opentelemetry/constants.py

```diff
@@ -0,0 +1,56 @@
+INPUT_KEYS = {
+    "value": [
+        "weave.input",  # Weave
```

Comment:
hmmm - how does weave get in here?

Reply:
This is addressed in https://github.com/wandb/weave/pull/4181 I'd review it first